logserver/src/main/java/alluxio/logserver/LoggingEventSubclass.java

```diff
@@ -0,0 +1,11 @@
+package alluxio.logserver;
+
+import org.apache.log4j.Category;
+import org.apache.log4j.Priority;
+import org.apache.log4j.spi.LoggingEvent;
+
+public class LoggingEventSubclass extends LoggingEvent {
+    public LoggingEventSubclass(String fqnOfCategoryClass, Category logger, Priority level, Object message, Throwable throwable) {
+        super(fqnOfCategoryClass, logger, level, message, throwable);
+    }
+}
```

Comment:
This subclass does not contain any new fields. What's your intention of including such a subclass?
Also, for such a simple test class that is only used in a single place, you can make it an inner class of the test class that uses it.